### PR TITLE
Sort favourite

### PR DESCRIPTION
**Pull request by Asilo5**
Attempted to toggle favourites on click but i cant seem to get it to work

i wrote out a reducer and action for it

as well as added a new property for the currentUser that could be able to store favourites into its own array

all most of the tests are failing so we can work on that in the weekend

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -30,21 +30,36 @@ class App extends Component {
     console.log(currentUser)
   }
 
+  toggleFavourites = (id) => {
+    let { movies, currentUser, favourite } = this.props;
+    let movie =  movies.find(movie => movie.id === id);
+    if (favourite === true) {
```
Comment (Asilo5) [inline src/Components/App/App.js]: Bless your soul, Sam. This is great!! idk why i didn't think about this last night!! thank you!!

File: src/Components/MoviesContainer/__snapshots__/MoviesContainer.test.js.snap [hunk 2]
```diff
@@ -1,24 +1,3 @@
...
-    id={123}
-    key="123"
-    popularity={462.91}
-    title="Harry Potter and the Sorceres Stone"
-    video={false}
-    vote_count={3775}
-  />
-</section>
-`;
+exports[`MoviesContainer should match snapshot with all data being passed in correctly 1`] = `undefined`;
```
Comment (Asilo5) [inline src/Components/MoviesContainer/__snapshots__/MoviesContainer.test.js.snap]: Can you let me know later what this line does later exactly?

**Comment by SamuelColeman** [comment]
Im having an issue with the favourite changing from true to false, its working correctly and so are my conditionals but if I click on a new movie it will change from true to false, rather than starting at false and changing to true. We may need to have our favourite boolean somewhere else so it can be specific to each movie. Let me know if yall have any questions.

**Comment by Asilo5** [comment]
We can take a look at it, I'll merge it now.

**Pull request merged**